host-os-build.py

```diff
@@ -34,6 +34,7 @@ def main(args):
     packages_to_build = (CONF.get('default').get('packages')
                          or config.discover_software())
 
+    LOG.info("Building packages: %s", ", ".join(packages_to_build))
```

Comment:
I think they are useful at this point because the scheduler is only created after loading all the packages. Since we currently download sources during packages loading and that may take a long time, it is good to know beforehand if we passed the correct packages as parameters.
Besides, another patch replaces BuildManager with PackagesManager in the upgrade_versions script, and it does not use the scheduler.